stan/math/prim/prob/neg_binomial_lccdf.hpp

```diff
@@ -93,7 +93,7 @@ return_type_t<T_shape, T_inv_scale> neg_binomial_lccdf(
     const T_partials_return p_dbl = beta_dbl * inv_beta_p1;
     const T_partials_return d_dbl = square(inv_beta_p1);
     const T_partials_return Pi = 1.0 - inc_beta(alpha_dbl, n_dbl + 1.0, p_dbl);
-    const T_partials_return beta_func = stan::math::beta(n_dbl + 1, alpha_dbl);
+    const T_partials_return beta_func = beta(n_dbl + 1, alpha_dbl);
```

Comment:
[question]
Any idea why the right version wasn't getting picked up with the qualifier?  It should have all the candidates in scope as long as they've been included and if they haven't been included, I don't see how the unqualified version could work.